app/views/articles/_form.html.erb

```diff
@@ -25,12 +25,13 @@
 </p>
 
 <p>
-  <%= select_tag(:category_id, options_for_select(Category.all.map{|c| [ c.name, c.id ] }), :prompt => "Select one!") %>
+  <%= select_tag(:category_id, options_for_select(Category.all.map{|c| [ c.name, c.id ] }, selected: @article.category_id),
+                  prompt: "Select one!") %>
 </p>
 
 
 <p>
-  <%= form.submit %>
+  <%= form.submit class: 'button' %>
 </p>
 
 <% end %>

```

Comment:
It would be good practice to indent all of the elements that occur within the opening of the form tag (`form_with`) and its end (here)